onnxruntime/python/tools/transformers/benchmark_helper.py

```diff
@@ -164,9 +164,11 @@ def prepare_environment(cache_dir, output_dir, use_gpu, provider=None):
             ), "Please install onnxruntime-directml package to test GPU inference."
 
         else:
-            assert (
-                "CUDAExecutionProvider" in onnxruntime.get_available_providers()
-            ), "Please install onnxruntime-gpu package to test GPU inference."
+            assert any(
```

Comment:
You forgot to remove the any() here. `isdisjoint` have already returned a `bool` here.

Reply:
Ah right. Let me fix this